vfs.go

```diff
@@ -485,7 +485,7 @@ func (v *VFSfh) Close() error {
 }
 
 // Read part of a file
-func (v *VFSfh) Read(p []byte) (uint64, error) {
+func (v *VFSfh) Read(p []byte) (int, error) {
```

Comment:
Why are we changing the return type here since all usage in function is of uint64?

Reply:
This was because of the signature of the reader interface: https://golang.org/pkg/io/#Reader

It was a mistake for me to have changed this previously

Reply:
👍